config/500-webhook.yaml

```diff
@@ -24,12 +24,12 @@ spec:
   selector:
     matchLabels:
       app: webhook
```

Comment:
Is this label needed or wanted?

Reply:
yes: https://github.com/knative/serving/blob/master/config/webhook.yaml#L22-L36

Reply:
The pair of labels goes all the way back to the [initial commit](https://github.com/knative/serving/blob/7f5da0b7438f562ea4a45c7021a674e2f90a5ddf/webhook.yaml#L25-L26), but that doesn't answer why it is there. Does anything look at the `app` label and do something special with it?

Reply:
I think that is how the rp and the pods know about eachother?